generators/spring-boot/resources/spring-boot-dependencies.pom

```diff
@@ -702,26 +635,96 @@
         <artifactId>elasticsearch-java</artifactId>
         <version>${elasticsearch-client.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.flywaydb</groupId>
+        <artifactId>flyway-commandline</artifactId>
+        <version>${flyway.version}</version>
+      </dependency>
       <dependency>
         <groupId>org.flywaydb</groupId>
         <artifactId>flyway-core</artifactId>
         <version>${flyway.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.flywaydb</groupId>
+        <artifactId>flyway-database-db2</artifactId>
+        <version>${flyway.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.flywaydb</groupId>
+        <artifactId>flyway-database-derby</artifactId>
+        <version>${flyway.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.flywaydb</groupId>
+        <artifactId>flyway-database-hsqldb</artifactId>
+        <version>${flyway.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.flywaydb</groupId>
+        <artifactId>flyway-database-informix</artifactId>
+        <version>${flyway.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.flywaydb</groupId>
+        <artifactId>flyway-database-mongodb</artifactId>
+        <version>${flyway.version}</version>
+      </dependency>
```

Comment:
Do we need these Flyway dependencies? We use Liquibase so they seem unnecessary.

Reply:
Nevermind, I think this is just the Spring Boot dependencies POM that's copied into the project.

Reply:
Yes, we have the spring-dependencies bom embedded so we can get information from it.
We use its properties.